### DISCUSSION TITLE
clipPath mask about outside border

### DISCUSSION THREAD

**Discussion by maoweifan** [fabricjs/fabric.js#7899] [Q&A]
## Version
5.2.1

<img width="753" alt="image" src="https://user-images.githubusercontent.com/51221758/164589677-68d421fc-e13f-4be7-a629-29e9f2cf300c.png">

I added a clipPath to the Rect (used as a mask)
I want to display the border of fabric.Object outside the mask
What should I do?

**Comment by ShaMan123** [comment]
It's not explained good enough.
Add a working reproduction.
What is the value of `preserveObjectStacking`?
